src/main/java/com/dataguise/saas/util/Utility.java

```diff
@@ -0,0 +1,114 @@
+package com.dataguise.saas.util;
+
+import com.dg.saas.orch.models.structures.ErrorConstants;
+import com.google.gson.JsonObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+
+public class Utility {
```

Comment:
**Reporter**: PMD    
    **Rule**: ClassNamingConventions
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/util/Utility.java L20
    
    The utility class name &#39;Utility&#39; doesn&#39;t match &#39;[A-Z][a-zA-Z0-9]+(Utils?|Helper)&#39;&#10;&#10;&#10;Code Style https://pmd.github.io/pmd/pmd_rules_java_codestyle.html#classnamingconventions
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1860569370>*